Pipfile.lock

```diff
@@ -179,68 +194,68 @@
         },
         "urllib3": {
             "hashes": [
-                "sha256:8d36afa7616d8ab714608411b4a3b13e58f463aee519024578e062e141dce20f",
-                "sha256:8f135f6502756bde6b2a9b28989df5fbe87c9970cecaa69041edcce7f0589b14"
+                "sha256:c97dfde1f7bd43a71c8d2a58e369e9b2bf692d1334ea9f9cae55add7d0dd0f84",
+                "sha256:fdb6d215c776278489906c2f8916e6e7d4f5a9b602ccbcfdf7f016fc8da0596e"
             ],
-            "markers": "python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4, 3.5'",
-            "version": "==1.26.16"
+            "markers": "python_version >= '3.7'",
+            "version": "==2.0.7"
```

Comment:
Given this major version change for `urllib3` from `1.26.16` to `2.0.7`, assuming all the tests ran okay?

Reply:
As noted in slack, not seeing anything in the `2.x` changelog that would suggest this version bump would be problematic: https://github.com/urllib3/urllib3/blob/main/CHANGES.rst#L88-L114.

Reply:
all of the extra integration tests passed